src/internal/connector/graph_connector.go

```diff
@@ -338,5 +351,5 @@ func getResources(
 		return nil, clues.Stack(err).WithClues(ctx).WithAll(graph.ErrData(err)...)
 	}
 
-	return resources, errs.ErrorOrNil()
+	return resources, errs.Err()
```

Comment:
this is going to hide non-catastrophic errors which is different than the current behavior. That's what we want in the long run, but for right now it's not clear if this will end up hiding errors from us especially since `errs.Errs()` is never checked in the cli code (or other code in this PR it seems)